src/app/pages/ArticlePage/ArticlePage.tsx

```diff
@@ -119,10 +125,52 @@ const DisclaimerWithPaddingOverride = (props: ComponentToRenderProps) => (
 const getPodcastPromoComponent = (podcastPromoEnabled: boolean) => () =>
   podcastPromoEnabled ? <InlinePodcastPromo /> : null;
 
+const getHeadlineComponent =
```

Comment:
Should we consider moving this into its own file next to the Article Page? We do this for the election banner, experiments & page promo sections...